app/src/main/java/org/sopt/hyundaicarddive/presentation/ui/detail/component/DetailMenu.kt

```diff
@@ -0,0 +1,75 @@
+package org.sopt.hyundaicarddive.presentation.ui.detail.component
+
+import androidx.compose.foundation.Image
+import androidx.compose.foundation.layout.Arrangement
+import androidx.compose.foundation.layout.Column
+import androidx.compose.foundation.layout.Row
+import androidx.compose.foundation.layout.fillMaxWidth
+import androidx.compose.foundation.layout.padding
+import androidx.compose.material3.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.res.painterResource
+import androidx.compose.ui.res.stringResource
+import androidx.compose.ui.tooling.preview.Preview
+import androidx.compose.ui.unit.dp
+import org.sopt.hyundaicarddive.R
+import org.sopt.hyundaicarddive.ui.theme.HYUNDAICARDDIVETheme
+
+enum class DetailMenuItem(
+    val iconRes: Int,
+    val stringRes: Int
+) {
+    SPACE(R.drawable.ic_spacedetail_space_28, R.string.detail_menu_space_title),
```

Comment:
명확히 어떤 속성에 어떤 값이 들어가는지 해주는게 좋을 것 같아요 ~
```suggestion
    SPACE(
    iconRes = R.drawable.ic_spacedetail_space_28,
    stringRes = R.string.detail_menu_space_title
    ),
```